tests/Resources/ti.network.httpclient.test.js

```diff
@@ -255,9 +255,9 @@ describe('Titanium.Network.HTTPClient', function () {
 			if (xhr.status === 200) {
 				should(e.success).be.true();
 
-				const response = JSON.parse(xhr.responseText);
-				response['adhocHeader'].should.eql('notcleared');
-				response.should.not.have.property('clearedHeader');
+				const response = JSON.parse(xhr.responseText).headers;
+				response['Adhoc-Header'].should.eql('notcleared');
```

Comment:
httpbin enforces casing on the headers so I just made them look like normal headers